tensorflow/training/docker/2.9/py3/cu112/Dockerfile.gpu

```diff
@@ -405,6 +405,9 @@ ENV TF_AUTOTUNE_THRESHOLD=2
 
 ENV LD_LIBRARY_PATH="/usr/local/lib/${PYTHON}/site-packages/smdistributed/dataparallel/lib:$LD_LIBRARY_PATH"
 
+# Remove python kernel installed by sparkmagic
```

Comment:
Was there a specific reason this was added now?

Reply:
Removing ipykernel to ensure proper usage in SM Studio